src/site/shared/containers/HistoryBox/index.tsx

```diff
@@ -33,7 +36,8 @@ const GroupActionEntry = ({g}: GroupActionEntryProps) => {
     return (
         <div className="historybox__groupentry">
             <span>Group Action</span>
-            {g.getActions().map((a, i) => {
+            <span className="historybox__groupentrycollapse" onClick={() => setIsCollapsed(!isCollapsed)}>&rsaquo;</span>
```

Comment:
nit: call it `historybox__groupentry-collapse` (and edit the corresponding SCSS name as well)